src/App.js

```diff
@@ -28,5 +28,37 @@ Feel free to have fun and experiment with others too!
 If you get stuck or need help, please reach out on Slack.
 */
 export default function App() {
-  return <div>Replace this with your beautiful JSX</div>;
+  return (
+    <div className="App">
```

Comment:
As an alternative, this could be a good instance to use a [fragment](https://reactjs.org/docs/fragments.html) since there's no required styling for this section - but a `div` also works!